{{cookiecutter.project_dir}}/{{cookiecutter.project_dir}}-ui/server.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const path = require("path");
+const app = express();
+const port = 80;
+
+const buildFolder = "./build";
+
+const { API_HOST } = process.env;
+
+app.set("views", path.join(__dirname, buildFolder));
+app.engine("html", require("ejs").renderFile);
+
+app.use(
+  "/static",
+  express.static(path.join(__dirname, `${buildFolder}/static`))
+);
+
+app.use(
+  "/images",
+  express.static(path.join(__dirname, `${buildFolder}/images`))
+);
+
+app.use("/__health", (req, res) => {
+  res.setHeader("Content-Type", "application/json");
+  res.send({ status: "ok" });
+});
+
+app.get("/*", function (req, res) {
```

Comment:
there should be method added here which calls the server's translate API. We should not simply be passing the env variable to the page.